src/lib.rs

```diff
@@ -20,7 +20,7 @@ extern crate rustc;
 
 #[cfg(unix)]
 extern crate libc;
-extern crate test;
+extern crate libtest as test;
```

Comment:
Which `test` did this link against before? Clearly something that worked on stable, whereas `libtest` does not compile on stable because it depends on `rustc_term`.

Reply:
Oh, I think it was https://github.com/messense/rustc-test.

Reply:
Rustc ships with a libtest